internal/venice-client-common/src/main/java/com/linkedin/venice/stats/VeniceOpenTelemetryMetricsRepository.java

```diff
@@ -80,62 +83,75 @@ public VeniceOpenTelemetryMetricsRepository(VeniceMetricsConfig metricsConfig) {
         metricsConfig.toString());
     this.metricPrefix = metricsConfig.getMetricPrefix();
     validateMetricName(getMetricPrefix());
-    try {
-      SdkMeterProviderBuilder builder = SdkMeterProvider.builder();
-
-      if (metricsConfig.exportOtelMetricsToEndpoint()) {
-        MetricExporter httpExporter = getOtlpHttpMetricExporter(metricsConfig);
-        builder.registerMetricReader(
-            PeriodicMetricReader.builder(httpExporter)
-                .setInterval(metricsConfig.getExportOtelMetricsIntervalInSeconds(), TimeUnit.SECONDS)
-                .build());
+    if (metricsConfig.useOpenTelemetryInitializedByApplication()) {
+      LOGGER.warn("Configured to use GlobalOpenTelemetry set by the application");
```

Comment:
Nit, could it be removed as we have a log line at L94?

Reply:
yes moved around the logs.